src/pass/arg_binder.cc

```diff
@@ -184,7 +184,9 @@ void ArgBinder::BindDLTensor(const Buffer& buffer,
                UIntImm::make(UInt(8), dtype.bits()) &&
                TVMArrayGet(UInt(16), handle, intrinsic::kArrTypeLanes) ==
                UIntImm::make(UInt(16), dtype.lanes()));
-  asserts_.emplace_back(AssertStmt::make(cond, type_err_msg.str(), nop));
+  if (!(dtype == Int(4) || dtype == UInt(4) || dtype == Int(1))) {
```

Comment:
Sorry I have no idea about this hack. But can you please leave a todo to remind others to fix it later?

Reply:
This hack is added because int4/int1 is not supported in basic data types as in numpy or c, we can only feed int4/int1 data in int8 or int32，which is not aligned with tvm ir's data type Int(4) or Int(1) etc. I wonder if you guys have a better solution than this hacky one.